src/ui/AddPhotoBlock/AddPhotoBlock.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import './AddPhotoBlock.scss';
+
+const AddPhotoBlock = ({ photo, setPhoto, name, sizeLimit, labelText }) => {
+  const [errorText, setErrorText] = useState('');
+
+  const handlePhoto = (e) => {
+    const [loadingFile] = e.target.files;
```

Comment:
```suggestion
const loadingFile = e.target.files[0];
```
мы ожидаем только один выбранный файл, поэтому, возможно, не нужно деструктуризировать массив

Reply:
К сожалению, eslint вариант e.target.files[0] не принимает. В целом ничего страшного, так как деструктуризируем только первый элемент массива